lib/order.rb

```diff
@@ -0,0 +1,11 @@
+class Order
+
+  def initialize(menu = Menu.new)
```

Comment:
`(menu = Menu.new)` may not need to be there, on the next line you could have `@menu = Menu.new`